classes/Pack.php

```diff
@@ -463,7 +463,7 @@ public static function deleteItems($id_product)
      */
     public static function addItem($id_product, $id_item, $qty, $id_attribute_item = 0)
     {
-        $id_attribute_item = (int) $id_attribute_item ? (int) $id_attribute_item : Product::getDefaultAttribute((int) $id_item);
+        $id_attribute_item = (int) $id_attribute_item ? (int) $id_attribute_item : Product::getDefaultAttribute((int) $id_item, 0, true);
```

Comment:
Refresh the pack default attribute, or in behat tests Some previous combination values were cached and reused in later tests Which made them fail